ci-typing.py

```diff
@@ -31,24 +31,24 @@
 #
 #
 
-set -euo pipefail
+from __future__ import annotations
 
-ruff_fix=("--fix")
-if [[ ${1-} == "--check" ]]; then
-    ruff_fix=("--diff")
-    black_options=("--check" "--diff")
-    shift
-fi
+import subprocess
+import sys
 
-files=("$@")
 
-if [[ $# == 0 ]]; then
-    files=(".")
-fi
+def main() -> None:
+    try:
+        subprocess.run(
+            ["poetry", "run", "mypy", "src", "tests/unit"],  # noqa: S603, S607
```

Comment:
Should be `"tests"` instead of `"tests/unit"` with an appropriate `exclude`/`ignore` for the resourses